tests/infra/remote.py

```diff
@@ -135,12 +135,16 @@ def __init__(
         self.files += data_files
         self.cmd = cmd
         self.client = paramiko.SSHClient()
+        # this client is used to execute commands on the remote host since the main client uses pty
```

Comment:
I assume the comment refers to the new `self.proc_client`. Could it be made clearer?